### PR TITLE
Phylotree

### PR DESCRIPTION
**Pull request by kwendel**
The phylotree is now displayed and the user can select multiple genomes or a common ancestor

### CODE REVIEW

**Comment by Delfil** [comment]
http://g2f.nl/00u32k4
Swing issue?

**Comment by Delfil** [comment]
Is there a link to the project of the newick parser?
Everything looks fine.

File: src/main/java/tudelft/ti2806/pl3/FileSelector.java [hunk 1]
```diff
@@ -57,4 +65,28 @@ public String getDescription() {
 		}
 		return null;
 	}
+
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/FileSelector.java]: Een parser in FileSelector? Dat is raar :P

Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/FileSelector.java]: Do you mean the .replaceAll lines? I placed them in Application originally but Kasper moved them to FileSelector.

**Comment by borismattijssen** [comment]
All good now

**Comment by TomBrouws** [comment]
Looks good!

**Pull request merged**